_conferences/droidcon-vietnam-2023.md

```diff
@@ -0,0 +1,13 @@
+---
+name: "Droidcon"
+website: https://www.vietnam.droidcon.com/
+location: Ho Chi Minh City, Vietnam
+
+date_start: 2023-07-15
+date_end:   2023-07-16
```

Comment:
According to the website it's a one day event. So `date_end` should be `2023-07-15` as well.

Reply:
Good catch, I just copied that part off the Sessionize page which might be representing the conference wrong

Reply:
I just talked to one of the organizers and they updated the Sessionize page.